File: js/fibonacci.js
const fibonacci = (num) => {
    let num1=0;
    let num2=1;
    let sum=0;

      for(i=0;i<num;i++){
        sum = num1 +num2;
        num1 = num2;
        num2 = sum;
      }
      return num1    
}

module.exports = {fibonacci}

File: python/fibonacci.py
def fibonacci(n):
  num1 = 0
  num2 = 1
  sum = 0

  if n < 1 : 
    return 1

  i=0
  for i in range(0,n) :
    sum = num1 + num2
    num1 = num2
    num2 = sum 
    i+1
   

  return num1    


